t/plugin/proxy-rewrite3.t

```diff
@@ -328,12 +379,95 @@ passed
 
 
 
-=== TEST 13: rewrite X-Forwarded-Host
+=== TEST 15: add exist header in muti-header
 --- request
 GET /echo HTTP/1.1
 --- more_headers
-X-Forwarded-Host: apisix.ai
+test: sssss
+test: bbb
 --- response_headers
-X-Forwarded-Host: test.com
+test: sssss, bbb, 123
+--- no_error_log
```

Comment:
Need to deal with https://github.com/apache/apisix/pull/8336#discussion_r1026081925 in other test

Reply:
done